README.md

```diff
@@ -55,6 +55,9 @@ driver:
 The `image_ref` and `flavor_ref` options can be specified as an exact id,
 an exact name, or as a regular expression matching the name of the image or flavor.
 
+All of Fog's `openstack` options (`openstack_domain_name`, `openstack_project_name`,
```

Comment:
How's this sound?

I would've liked to have a link to a definitive list of options, but I couldn't find one in the docs. We could link to the code, I guess.